src/ddl_utils.c

```diff
@@ -0,0 +1,72 @@
+#include <postgres.h>
+#include <tcop/deparse_utility.h>
+
+/*
+ * Test if ddl command as an alter table add column
+ */
+PG_FUNCTION_INFO_V1(ddl_is_add_column);
+Datum
+ddl_is_add_column(PG_FUNCTION_ARGS)
```

Comment:
Should we go one step further here and instead of returning just a bool, return the attnum or somesuch to make the above sql code even less error-prone? (applies to ddl_is_drop_column too). Also should we do this for all ddl commands?